code/modules/vehicles/scooter.dm

```diff
@@ -24,19 +24,44 @@
 	if(buckled_mobs.len)
 		for(var/m in buckled_mobs)
 			var/mob/living/buckled_mob = m
-			switch(buckled_mob.dir)
-				if(NORTH)
-					buckled_mob.pixel_x = 0
-					buckled_mob.pixel_y = 4
-				if(EAST)
-					buckled_mob.pixel_x = -2
-					buckled_mob.pixel_y = 4
-				if(SOUTH)
-					buckled_mob.pixel_x = 0
-					buckled_mob.pixel_y = 4
-				if(WEST)
-					buckled_mob.pixel_x = 2
-					buckled_mob.pixel_y = 4
+			if(buckled_mobs.get_num_legs() > 0)
```

Comment:
this should be buckled_mobs not buckled_mob

this is now the second time you have done this :laughing: